NHLBI_BioData_Catalyst/Rstudio/4_Genomic_Queries.Rmd

```diff
@@ -51,31 +51,33 @@ session <- picsure::bdc.setResource(session = session,  resourceName = "AUTH")
 # Example 1: Build a query with genomic and phenotypic variables
 ## 1.1 Building the query with the PIC-SURE API
 
-We are going to create a new query request from the PIC-SURE resource that was specified above.  For this example, we will limit the query to a single study, a single phenotype (gender and age range), and two genomic filters. 
+We are going to create a new query request from the PIC-SURE resource that was specified above.  For this example, we will limit the query to a single dataset (the TOPMed DCC Harmonized dataset) by filtering on sex, body mass index, and two genomic filters. 
+
+For more information about the TOPMed DCC Harmonized dataset, please refer to the [`2_TOPMed_DCC_Harmonized_Variables_analysis.ipynb` notebook](./2_TOPMed_DCC_Harmonized_Variables_analysis.ipynb).
 
 First we will create a new query instance.
 
 ```{r}
 my_query <- picsure::bdc.newQuery(session) # Initialize a new query
 ```
 
-### Find all SAGE variables
+### Find all TOPMed DCC Harmonized variables
 We can search for variables related to our search query using the `bdc::find.in.dictionary` function. 
```

Comment:
I don't think this is the function used any more, isn't this searchPicsure now?

Reply:
Great catch! thanks!